src/common/components/cards/card-footer-instance-action-buttons.tsx

```diff
@@ -38,102 +48,119 @@ export interface CardFooterInstanceActionButtonsProps {
-            // the dialog/toast involve to be considered as part of the button for the purposes of layout
-            // calculation in this component's parent.
-            <div onKeyDown={event => event.stopPropagation()}>
-                {this.renderButtons()}
-                {this.renderCopyFailureDetailsToast()}
-            </div>
-        );
-    }
+        const getMenuItems = () => {
+            return props.deps.cardFooterMenuItemsBuilder.getCardFooterMenuItems({
+                ...props,
+                toastRef: toastRef,
+                fileIssueButtonRef: ref => (fileIssueButtonRef.current = ref),
+                onIssueFilingSettingsDialogDismissed: focusButtonAfterDialogClosed,
+            });
+        };
 
-    public renderButtons(): JSX.Element {
-        if (this.props.narrowModeStatus?.isCardFooterCollapsed) {
-            return this.renderKebabButton();
-        } else {
-            return this.renderExpandedButtons();
-        }
-    }
+        const renderCopyFailureDetailsToast = () => {
+            const { cardInteractionSupport } = props.deps;
 
-    public renderKebabButton(): JSX.Element {
-        return (
-            <ActionButton
-                componentRef={ref => (this.kebabButtonRef = ref)}
-                ariaLabel={this.props.kebabMenuAriaLabel || 'More actions'}
-                menuIconProps={{
-                    iconName: 'MoreActionsMenuIcon',
-                    className: styles.kebabMenuIcon,
-                }}
-                menuProps={{
-                    directionalHint: DirectionalHint.bottomRightEdge,
-                    shouldFocusOnMount: true,
-                    items: this.getMenuItems(),
-                }}
-            />
-        );
-    }
+            if (!cardInteractionSupport.supportsCopyFailureDetails) {
+                return null;
+            }
 
-    public renderExpandedButtons(): JSX.Element {
-        const menuItems = this.getMenuItems();
+            if (props?.narrowModeStatus?.isCardFooterCollapsed) {
+                kebabButtonRef?.current?.focus();
+            }
 
-        return (
-            <>
-                {menuItems.map(props => (
-                    <span key={props.key}>
-                        <ActionButton
-                            onClick={props.onClick}
-                            text={props.text}
-                            iconProps={props.iconProps}
-                            className={props.key}
-                            componentRef={props.componentRef}
-                        />
-                    </span>
-                ))}
-            </>
-        );
-    }
+            return <Toast ref={toastRef} deps={props.deps} />;
+        };
 
-    public renderCopyFailureDetailsToast(): JSX.Element | null {
-        const { cardInteractionSupport } = this.props.deps;
+        const renderKebabButton = () => {
```

Comment:
Hi @madalynrose this warning is coming for command bar buttons menu because of misplacement of tooltip, fixed, now the warning is not coming, please review

Reply:
Okay, but there is still a bug in the interaction of this element that needs to be resolved:
https://microsoft-my.sharepoint.com/:v:/p/madparker/EVmeVAx2bwpNoBj4bwdp5gAB1r9lEcVqTZXpUP58dQQErg?e=bIwWEa

Reply:
Hi @madalynrose yeah i have fixed this, it was happening because on render we are calling renderCopyFailureDetailsToast method which had kebabButtonRef?.current?.focus(); this was causing focus to got to other expanded section kebab button. I have tested working fine now. Please re-review